_docs/setup/kubernetes/multicluster-install.md

```diff
@@ -120,7 +106,7 @@ The istio-remote component must be deployed to each remote Kubernetes cluster.
 1. Use the helm template command on a remote to specify the Istio control plane:
 
    ```bash
-   helm template install/kubernetes/helm/istio-remote --name istio-remote --set pilotEndpoint=`pod_ip_of_pilot_master` --set mixerEndpoint=`pod_ip_of_mixer_master` > $HOME/istio-remote.yaml
+   helm template install/kubernetes/helm/istio-remote --name istio-remote --set pilotEndpoint=`pod_ip_of_pilot_local` --set policyEndpoint=`pod_ip_of_policy_local --set statsdEndpoint=`pod_ip_of_statsd_local` > $HOME/istio-remote.yaml
```

Comment:
What does `pod_ip_of_pilot_local`, `pod_ip_of_policy_local`, `pod_ip_of_statsd_local` means?

Is it mean the pod ip of those components in `local` control plane? Why use pod ip? How can we make sure the pod ip can be accessed cross clusters? How to handle the case if I have multiple pods for pilot?

Reply:
How can I make sure the pod ip in cluster1 can be accessed from cluster2?

Reply:
There is no service IP or ILB to use at this time, although that is planned for 0.9.  The pod IP is the only IP address we can currently reference to connect to pilot across clusters.  The requisites cover the requirements that pod networks are routable to one another.  I am not sure how to handle the case of multiple pods for pilot.  Is that something that can be deployed today?

In general I think all of your concerns here will be resolved once ILBs are available in the 0.9 implementation, however, I can't document the future.  Instead only documenting the current state of the system.

Reply:
The technique I use is as follows:

```
sdake@falkor-08:~$ cat route08.sh
#!/bin/bash
ip route add 10.111.0.0/16 via 10.23.220.75
```

And on my other cluster

```
sdake@falkor-07:~$ cat route07.sh
#!/bin/bash
ip route add 10.121.0.0/16 via 10.23.220.76
```

Clearly Kubernetes operators may have different network environments or may be using a VPN, and I don't think it is feasible to document how to setup routing on the pod IPs.

Reply:
partially clarified.

Reply:
Are you using calico or ovs for your cluster to configure the route?

Reply:
Do you know how can I enable this if I was using calico?

Reply:
@sdake any comments for this? Are you using ovs as an example? Any idea or suggestions if I was using calico for container network? How can I enabled pods communicate cross clusters.